circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  node:
+    version: 7
+  pre:
+    - mkdir ~/.yarn-cache
+deployment:
+  staging:
+    branch: master
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=staging
+  production:
+    branch: release
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=production
```

Comment:
No, it'd mean your deploy process is `git push upstream master:staging` or `git push upstream master:release`